.github/workflows/release.yaml

```diff
@@ -71,14 +71,23 @@ jobs:
         run: mv release ../
 
       - name: Checkout helm chart
+        if: ${{ github.event.inputs.step == '5' || github.event.inputs.step == '5-publish-helm' }}
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           persist-credentials: false
           repository: cilium/charts
           path: "./cilium-charts"
 
-      - name: Move helm charts to upper directory
-        run: mv cilium-charts ../charts
+      - name: Set up charts repository git configuration
+        if: ${{ github.event.inputs.step == '5' || github.event.inputs.step == '5-publish-helm' }}
+        shell: bash
+        run: |
+          mv cilium-charts ../charts
+          cd ../charts
```

Comment:
nit: Is it possible to check out the directory in the desired location in the first place so the mv is unnecessary?

Reply:
no, the checkout needs to happen under the workdir.